crates/rnote-ui/src/canvas/input.rs

```diff
@@ -119,7 +119,14 @@ pub(crate) fn handle_pointer_controller_event(
                 if gdk_button == gdk::BUTTON_PRIMARY {
                     pen_state = PenState::Up;
                 } else {
-                    pen_state = PenState::Proximity;
+                    #[cfg(target_os = "windows")]
```

Comment:
Consider using [cfg!](https://doc.rust-lang.org/stable/std/macro.cfg.html) and a `if {} else {}`